libraries/botbuilder-lg/src/generated/LGFileParserListener.ts

```diff
@@ -8,133 +8,133 @@
  * Licensed under the MIT License.
  */
 
-import { ParseTreeListener } from "antlr4ts/tree/ParseTreeListener";
+import { ParseTreeListener } from 'antlr4ts/tree/ParseTreeListener';
 
-import { FileContext } from "./LGFileParser";
-import { ParagraphContext } from "./LGFileParser";
-import { CommentDefinitionContext } from "./LGFileParser";
-import { ImportDefinitionContext } from "./LGFileParser";
-import { OptionDefinitionContext } from "./LGFileParser";
-import { ErrorDefinitionContext } from "./LGFileParser";
-import { TemplateDefinitionContext } from "./LGFileParser";
-import { TemplateNameLineContext } from "./LGFileParser";
-import { TemplateBodyContext } from "./LGFileParser";
-import { TemplateBodyLineContext } from "./LGFileParser";
+import { FileContext } from './LGFileParser';
+import { ParagraphContext } from './LGFileParser';
+import { CommentDefinitionContext } from './LGFileParser';
+import { ImportDefinitionContext } from './LGFileParser';
+import { OptionDefinitionContext } from './LGFileParser';
+import { ErrorDefinitionContext } from './LGFileParser';
+import { TemplateDefinitionContext } from './LGFileParser';
+import { TemplateNameLineContext } from './LGFileParser';
+import { TemplateBodyContext } from './LGFileParser';
+import { TemplateBodyLineContext } from './LGFileParser';
 
 
 /**
  * This interface defines a complete listener for a parse tree produced by
  * `LGFileParser`.
  */
 export interface LGFileParserListener extends ParseTreeListener {
-	/**
+    /**

```

Comment:
The comments on this file re not aligned

Reply:
Probably tabs instead of spaces, in the file looks fine.

Reply:
yep, seems file in the file.
Also, we could check the spacing between the comments and the previous method in that file